draft-ietf-rats-eat.md

```diff
@@ -1099,12 +1103,53 @@ string naming the submodule. No submodules may have the same name.
 {::include cddl/submods.cddl}
 ~~~~
 
+
 # Endorsements and Verification Keys {#keyid}
 
 TODO: fill this section in. It will discuss key IDs, endorsement ID and such that
 are needed as input needed to by the Verifier to verify the signature. This will
 NOT discuss the contents of an Endorsement, just and ID/locator.
 
+
+# Layered Attestation {#layered}
+
+It is possible for a device to have multiple Attesting Environments as described in {{RATS-Architecture}}.
+When these are layered, one is a root the endorsement and/or verification key originates with the device manufacturer or endorser.
+The trust for those that are layered above the root comes from Attestation Evidence about them that is provided by the root or a layer below.
+
+A layered Attesting Environment is always described as a submodule.
+To know that a particular submodule is providing evidence for an Attesting Environment, it must have an intended use claim with the value of "attest=env".
+See {{intended-use}}.
+
+The submodule should also contain a claim that hold the verification key, the key used to verify signed attestation evidence from the layered attester.
+A specific claim is defined to hold verification keys.
+There is no requirement on the algorithm or key type used by the layered attestation.
+There is a requirement that the included key is in the format of a COSE_key.
+
+There are two ways that the Verifier comes to trust a layered Attesting Environment.
+
+First, the root or lower layer Attesting Environment provides claims, perhaps many claims, characterizing the Attesting Environment.
+The Verifier recognized the claims are about an Attesting Environment by the intended use claim and uses the other claims to decide whether it trusts this layered Attesting Environment.
+The decision is made by the Verifier.
+
+Second, the root or lower layer Attesting environment provides the is-trusted claim with a value of true.
+This tells the Verifier that the Attesting Environment has been sufficiently evaluated on the device by the root or lower layer Attesting Environment.
+The decision is made by on the device.
```

Comment:
So these are "EAT options", right? In general, a third option is that the Verifiers can obtain Endorsements that are about the Attesting Environment, I think.

Reply:
1135 I don't see how it is possible for a lower layer to know or assert that it knows the upper layer is trusted without going through a recursive verification. However, verification is done by a verifier which is a complex function. It would be a mistake to assert that layering requires attesting environments to also be verifiers according to the RATS Arch. The is-trusted claim is actually an attestation result structure. Doesn't belong here IMO.